pkg/k8s/run.go

```diff
@@ -79,6 +82,38 @@ func run(ctx context.Context, s *scanner, opt cmd.Option, artifacts []*artifacts
 	return nil
 }
 
-func getArtifacts(ctx context.Context, cluster k8s.Cluster, namespace string) ([]*artifacts.Artifact, error) {
-	return trivyk8s.New(cluster).Namespace(namespace).ListArtifacts(ctx)
+func getArtifacts(ctx context.Context, args cli.Args, cluster k8s.Cluster, namespace string) ([]*artifacts.Artifact, error) {
+	trivyk8s := trivyk8s.New(cluster).Namespace(namespace)
+
+	if !args.Present() {
+		return trivyk8s.ListArtifacts(ctx)
+	}
+
+	kind, name, err := extractKindAndName(args)
+	if err != nil {
+		return nil, err
+	}
+
+	artifact, err := trivyk8s.GetArtifact(ctx, kind, name)
+	if err != nil {
+		return nil, err
+	}
+
+	return []*artifacts.Artifact{artifact}, nil
+}
+
+func extractKindAndName(args cli.Args) (string, string, error) {
+	switch args.Len() {
+	case 1:
+		s := strings.Split(args.Get(0), "/")
+		if len(s) != 2 {
+			return "", "", fmt.Errorf("can't parse arguments: %v", args.Slice())
```

Comment:
Could you use `xerrors.Errorf` to be aligned with other places? This is because `fmt.Errorf` still doesn't support stack trace that is really useful in user's bug reports. The stack trace helped us with debugging many times.